jest.config.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   preset: 'ts-jest',
+  testMatch: ["<rootDir>/test/jest/**/*.ts"],
```

Comment:
I think this would conflict with the npm scripts which specify `testMatch` on a per-npm-script basis but share this same jest config file. Were you having a problem running jest tests?

Reply:
> Were you having a problem running jest tests?

Indeed. It was the `npm run test:jest:watch` script that needed `testMatch` params. Removed this line `jest.config.js`